content/yaml-quick-start/building-a-unity-app.md

```diff
@@ -0,0 +1,492 @@
+---
+title: Building Unity mobile apps
+description: How to build Unity mobile apps with codemagic.yaml
+weight: 12
+
+---
+
+Unity is a cross-platform game engine developed by Unity Technologies. It can be used to create mobile applications that run on iOS and Android.
+
+Codemagic's Unity premium macOS base image has the Unity SDK, iOS and Android modules pre-installed. 
+
+This guide will show you how to configure a workflow that builds and publishes your app to TestFlight and Google Play using a **codemagic.yaml** configuration file. 
+
+{{<notebox>}}
+**Important!** Building Unity apps on Codemagic is currently in closed beta release. If you are interested in taking part in the beta, please contact us [here](https://codemagic.io/contact/)
+{{</notebox>}}
+
+## Adding your Unity project to Codemagic
+
+The apps you have available on Codemagic are listed on the Applications page. Click **Add application** to add a new app.
+
+1. On the Applications page, click **Set up build** next to the app you want to start building. 
+2. On the popup, select **Other** as the project type and click **Continue**.
+3. In your IDE, create a [`codemagic.yaml`](./yaml). This will be used to create our workflow.
```

Comment:
The **Set up build** button and project type selection appear only for older repos. If the app is newly added, the project type selection happens already earlier in the flow, and on the Applications page the app shows up with **Finish build setup**. 
Actually the whole section here (and in other guides) is outdated. Instead, I would list here the requirements for building Unity (not necessarily in this order):
- Add a codemagic.yaml to the root of the repo and link to the workflow before
- Set up webhooks
- Add the Unity specific variables / activate the licence (link to the relevant section)
etc